consent/strategy_default.go

```diff
@@ -983,6 +991,27 @@ func (s *DefaultStrategy) HandleOpenIDConnectLogout(ctx context.Context, w http.
 	return s.completeLogout(ctx, w, r)
 }
 
+func (s *DefaultStrategy) HandleHeadlessLogout(ctx context.Context, w http.ResponseWriter, r *http.Request, sid string) error {
+	loginSession, lsErr := s.r.ConsentManager().GetRememberedLoginSession(ctx, sid)
+
+	// This is ok (session probably already revoked), do nothing!
+	if lsErr != nil {
```

Comment:
Should we test if `errors.Is(err, x.ErrNotFound)`?

Reply:
Also, should we not still perform the backchannel logout in this case?

Reply:
👍 for the specific error check.

For backchannel logout, we need a subject, which is stored in the Login Session. If we cannot find the session, then there is no subject as well, and we risk breaking the OIDC compliance by skipping the subject claim.